JuliaInterface/src/JuliaInterface.c

```diff
@@ -811,18 +811,6 @@ static Obj FuncJuliaModule(Obj self, Obj name)
     return NewJuliaObj((jl_value_t *)julia_module);
 }
 
-// Sets the value of the julia identifier <name> to the julia value the julia
-// object GAP object <julia_val> points to.
-static Obj FuncJuliaSetVal(Obj self, Obj name, Obj julia_val)
```

Comment:
I am pretty much against removing this function without implementing it again. I use it often for debugging purposes. We could hide it using `_` or provide a julia implementation. A Julia implementation could look like
```
function set_global(name,val)
           name_sym = Symbol(name)
           eval(:($name_sym=$val))
       end
```

Reply:
Of course if you the function, we can keep it, but then there should be a test case for it. And perhaps a comment saying "this function is useful for debugging".